docs/openapi3.yaml

```diff
@@ -2047,6 +2080,13 @@ components:
           type: number
           description: The duration of the test in seconds.
           example: 60
+        score:
+          type: number
+          description: The score of report comapring to test benchmark
```

Comment:
The score of a report, calculated based on a benchmark and benchmark weights

Reply:
fixed